src/measure_theory/integrable_on.lean

```diff
@@ -89,6 +89,31 @@ protected lemma measurable.measurable_at_filter (h : measurable f) :
   measurable_at_filter f l μ :=
 h.ae_measurable.measurable_at_filter
 
+lemma ae_measurable_indicator_iff [has_zero β] {s} (hs : measurable_set s) :
+  ae_measurable f (μ.restrict s) ↔ ae_measurable (indicator s f) μ :=
```

Comment:
you should probably exchange the two sides of the equivalence, to make it coherent with the lemma name.